internal/resolver/mtaresolver.go

```diff
@@ -24,10 +24,13 @@ const (
 	missingPrefixMsg   = `could not resolve the value for the "~{%s}" variable; missing required prefix`
 )
 
+// EnvFileName is the default environment variables file name
+const EnvFileName = ".env"
```

Comment:
1. This const can be inside the const clause above (just put an empty line to separate it). Its name should reflect the fact it's the default name (e.g. `defaultEnvFileName`).
2. Why is it exported (starts with an uppercase letter)? Do you use it in another package?

Reply:
It does not have to be exported. I changed it now to start with lower case and inside the existing const